website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -19,12 +19,17 @@ function _downloadEvent(event, item, col) {
 }
 
 // Define Fangorn Button Actions
-function _fangornActionColumn (item, col) {
-    var self = this;
+function _dataverseDefineToolbar(item) {
```

Comment:
For consistency and reuseability, these can be refactored into components as well, such that addons can optionally define a `ToolbarButtons` component, e.g.

```
var DataverseToolbarButtons = {
    view: function(ctrl, args, children) {
        var item = args.item;
        //....
    }
}
Fangorn.config.dataverse = {
    ToolbarButtons: DataverseToolbarButtons
};
```